app/services/hawkular_proxy_service.rb

```diff
@@ -78,7 +78,7 @@ def data(query)
   rescue StandardError => e
     {
       :parameters => params,
-      :error      => ActionView::Base.full_sanitizer.sanitize(e.message) + " " + _("(Please check your Hawkular server)")
+      :error      => ActionView::Base.full_sanitizer.sanitize(e.to_s) + " " + _("(Please check your Hawkular server)")
```

Comment:
Thanks 👍 , we probably want to leave making the message better to a next PR :-) . In this PR we will only do the 1:1 safe change of "#message" -> "#to_s"  , according to the docs they are identical when message is defined.